src/device/pf_diag.c

```diff
@@ -720,22 +819,22 @@ int pf_diag_remove (
                    * the specifier PNET_DIAG_CH_SPEC_ALL_DISAPPEARS is used. */
                   /* Overwrite the item to remove with correct cr_properties for
                    * this diag message. */
+                  /* Why change USI? */
```

Comment:
What does this comment refer to?

Reply:
It was a note to myself, now properly removed.